cell_classification/segmentation_data_prep.py

```diff
@@ -86,7 +83,7 @@ def get_image(self, data_folder, marker):
         img = imread(os.path.join(data_folder, marker + ".tiff"))
         return img
 
-    def get_instance_masks(self, data_folder, cell_mask_key):
+    def get_instance_mask(self, data_folder):
```

Comment:
I would give this function a different name, since it returns both the instance and binary mask

Reply:
Renamed to `get_inst_binary_masks`